Files/Scripts/Options/The Sealed Palace.psm1

```diff
@@ -108,20 +109,47 @@ function ByteOptions() {
 
     if ($Redux.Colors.SetEquipment -ne $null) {
         if (IsColor $Redux.Colors.SetEquipment[0] -Not)   { ChangeBytes -Offset "B9D1A8" -Values @($Redux.Colors.SetEquipment[0].Color.R, $Redux.Colors.SetEquipment[0].Color.G, $Redux.Colors.SetEquipment[0].Color.B) } # Kokiri Tunic
-        if (IsColor $Redux.Colors.SetEquipment[1] -Not)   { ChangeBytes -Offset "B9D1AB"  -Values @($Redux.Colors.SetEquipment[1].Color.R, $Redux.Colors.SetEquipment[1].Color.G, $Redux.Colors.SetEquipment[1].Color.B) } # Goron Tunic
```

Comment:
Do not change this, it's not related to your changes

Reply:
You have since told me you'll do the spacing, so I understand.